core/sail/federation/src/main/java/org/eclipse/rdf4j/sail/federation/Federation.java

```diff
@@ -94,6 +100,18 @@ public void addMember(Repository member) {
 		members.add(member);
 	}
 
+	List<Repository> getMembers() {
```

Comment:
Ie, being package private (default) means it is accessible to AbstractFederationConnection but no subclasses of this non-final class unless they place themselves inside of this package.